Failed tests (10505): TestConvergence

The following test appears to have failed:

[#10505](https://circleci.com/gh/cockroachdb/cockroach/10505):

```
I1231 11:19:23.601339 517 gossip/client.go:75  closing client to node 4 (127.0.0.1:34462): gossip/client.go:246: client closed
I1231 11:19:23.601657 517 gossip/client.go:75  closing client to node 1 (127.0.0.1:60915): gossip/client.go:246: client closed
I1231 11:19:23.602421 517 gossip/client.go:75  closing client to node 1 (127.0.0.1:60915): read tcp 127.0.0.1:54441->127.0.0.1:60915: use of closed network connection
W1231 11:19:23.602812 517 rpc/server.go:309  rpc: write response failed: write tcp 127.0.0.1:60915->127.0.0.1:54441: use of closed network connection
W1231 11:19:23.603234 517 rpc/server.go:309  rpc: write response failed: write tcp 127.0.0.1:60915->127.0.0.1:54441: use of closed network connection
--- FAIL: TestConvergence (0.44s)
	convergence_test.go:33: expected a fully-connected network within 15 cycles; took 16
FAIL
FAIL	github.com/cockroachdb/cockroach/gossip	0.777s
=== RUN   TestParseResolverSpec
--- PASS: TestParseResolverSpec (0.00s)
=== RUN   TestGetAddress
--- PASS: TestGetAddress (0.00s)
PASS
ok  	github.com/cockroachdb/cockroach/gossip/resolver	1.022s
?   	github.com/cockroachdb/cockroach/gossip/simulation	[no test files]
=== RUN   TestKeySorting
--- PASS: TestKeySorting (0.03s)
=== RUN   TestMakeKey

```
Please assign, take a look and update the issue accordingly.

--- a/gossip/convergence_test.go
+++ b/gossip/convergence_test.go
@@ -45,5 +45,5 @@ func verifyConvergence(numNodes, maxCycles int, t *testing.T) {
 // actual production gossip code than seems worthwhile for a unittest.
 func TestConvergence(t *testing.T) {
 	defer leaktest.AfterTest(t)
-	verifyConvergence(10, 15, t)
+	verifyConvergence(10, 20, t)
 }

